packages/file_selector/file_selector_macos/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
-## NEXT
+## 0.9.0
 
+* **BREAKING CHANGE**: Methods that take `acceptedTypeGroups` now throw an
+  `ArgumentError` if any group is not a wildcard (all filter types null or
+  empty), but doesn't include any of the filter types supported by macOS.
 * Ignores deprecation warnings for upcoming styleFrom button API changes.
```

Comment:
Should the ignore be landed as 0.8.2+3 so users aren't forced to upgrade a major version to not be affected by this lint? (does this even affect end users?)

Reply:
Nope, it doesn't. It's only in the changelog because I'm still figuring out better policies for when we skip changelogs in the new tool-enforced world.